### ISSUE TITLE
create new issue does not work

### ISSUE THREAD

**Issue by caleb15** [gioboa/jira-plugin#86]
**Describe the issue**
I can't create a new issue. When I click on insert issue the menu just flashes.

**To Reproduce**
1. Go to jira extension
2. Click on +
3. fill out required fields
4. Click Insert issue

**Expected behavior**
Some indication that issue is inserted

**Screenshots**
https://puu.sh/CLHsW/e182a2febc.png

How new issue screen looks like in chrome for comparison:
https://puu.sh/CLHAV/fba54ed5ce.png
https://puu.sh/CLHCd/b098f32351.png

The difference is that vscode does not show the required "department" field

**Desktop (please complete the following information):**
vscode info:

Version: 1.31.0 (system setup)
Commit: 7c66f58312b48ed8ca4e387ebd9ffe9605332caa
Date: 2019-02-05T22:35:56.624Z
Electron: 3.1.2
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17134

I have the latest version of the extension - I just installed it today.

**Smartphone (please complete the following information):**
 - ............. I don't think that is relevant.

**Additional context**

**Comment by caleb15** [comment]
oh and no errors are logged to the console.

**Comment by gioboa** [comment]
Hi @caleb15 ,
You got the problem, department field it's required but not present in list.
I need to understand how is department field schema. :thinking: 
#34 you must find something, there is a log in the code for this situation.

**Comment by caleb15** [comment]
you're right.  Here is the log:

```
{
    "required": true,
    "schema": {
        "type": "option",
        "custom": "com.atlassian.jira.plugin.system.customfieldtypes:select",
        "customId": 10031
    },
    "name": "Department",
    "key": "customfield_10031",
    "hasDefaultValue": false,
    "operations": [
        "set"
    ],
    "allowedValues": [
        {
            "self": "https://15five-dev.atlassian.net/rest/api/2/customFieldOption/10020",
            "value": "Support",
            "id": "10020"
        },
        {
            "self": "https://15five-dev.atlassian.net/rest/api/2/customFieldOption/10021",
            "value": "Product",
            "id": "10021"
        },
        {
            "self": "https://15five-dev.atlassian.net/rest/api/2/customFieldOption/10024",
            "value": "Engineering",
            "id": "10024"
        },
        {
            "self": "https://15five-dev.atlassian.net/rest/api/2/customFieldOption/10104",
            "value": "QA",
            "id": "10104"
        },
        {
            "self": "https://15five-dev.atlassian.net/rest/api/2/customFieldOption/10022",
            "value": "Sales",
            "id": "10022"
        },
        {
            "self": "https://15five-dev.atlassian.net/rest/api/2/customFieldOption/10023",
            "value": "CS",
            "id": "10023"
        },
        {
            "self": "https://15five-dev.atlassian.net/rest/api/2/customFieldOption/10025",
            "value": "Marketing",
            "id": "10025"
        },
        {
            "self": "https://15five-dev.atlassian.net/rest/api/2/customFieldOption/10026",
            "value": "Billing",
            "id": "10026"
        },
        {
            "self": "https://15five-dev.atlassian.net/rest/api/2/customFieldOption/10027",
            "value": "Other",
            "id": "10027"
        }
    ],
    "hideField": true
}
```

**Comment by gioboa** [comment]
Thanks, with this info I can fix the "insert issue algorithm". 
Soon I will release a new version with this hotfix. :+1:

**Comment by gioboa** [comment]
I released [v 0.13.2[(https://github.com/gioboa/jira-plugin/releases/tag/0.13.2) this version will solve you issue.
Please test it and give me you feedback. Thanks.

**Comment by caleb15** [comment]
It works :D

Thanks!
